pem-to-jwk/pom.xml

```diff
@@ -59,11 +59,18 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+  <pluginRepositories>
+    <pluginRepository>
+      <id>maven-snapshots</id>
+      <url>https://repository.apache.org/content/repositories/snapshots</url>
+    </pluginRepository>
+  </pluginRepositories>
   <build>
     <plugins>
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-shade-plugin</artifactId>
+        <version>3.3.0-SNAPSHOT</version>
```

Comment:
The latest released version of `maven-shade-plugin` does not support Java 16... using the latest SNAPSHOT is a gross workaround, as described [here](https://stackoverflow.com/questions/67997715/maven-compiler-plugin-unsupported-class-file-major-version-60)